gap/OscarInterface/gap/OscarInterface.gi

```diff
@@ -0,0 +1,45 @@
+InstallOtherMethod( ImagesRepresentative,
```

Comment:
```suggestion
DeclareAttribute( "JuliaData", IsObject );

InstallOtherMethod( ImagesRepresentative,
```

Reply:
I forgot to `git add gap/OscarInterface/gap/OscarInterface.gd`